mist-common/src/main/java/edu/snu/mist/common/operators/JoinOperator.java

```diff
@@ -113,4 +114,9 @@ public void processLeftData(final MistDataEvent input) {
   public void processLeftWatermark(final MistWatermarkEvent input) {
     outputEmitter.emitWatermark(input);
   }
+
+  @Override
+  public void processLeftCheckpoint(final MistCheckpointEvent input) {
+    outputEmitter.emitCheckpoint(input);
```

Comment:
How to checkpoint the states of  join and union operator?

Reply:
I don't think the state of join operator needs to be checkpointed.
The `joinBiPredicate`'s purpose does not require a state.